scripts/EN/Harry

```diff
@@ -1988,9 +1988,9 @@ LoadSecondEffect(0x0, 0x0, 0x0); // 0x23a8: 0000306e
 CleanScreen(); // 0x23b2: 00000064
 LoadBackground(0x16); // 0x23b6: 00001069
 DrawBackground(); // 0x23bc: 00000065
-ShowText((1107)"My one shot hits the wing, blasting a hole in it. The creature loses altitude as it falls to the floor."); // 0x23c0: 0000100c
+ShowText((1107)"My one shot hits the wing, W1blasting a large hole in it.¥nW1The creature then loses altitude and falls to the floor with a smack."); // 0x23c0: 0000100c
 HandleInput(); // 0x23c6: 00000010
-ShowText((1108)"I can't afford to waste bullets. I crush it with the with the thick sole of my shoe many times. As its strange cries die out, a dark red stain spreads out on the floor."); // 0x23ca: 0000100c
+ShowText((1108)"I can't afford to waste bullets.¥nW2I stomp on it with the thick sole of my shoe many times.¥nW2As its strange cries die out, W1a dark red stain spreads out on the floor."); // 0x23ca: 0000100c
```

Comment:
Removed an extra instance of "with the". Thanks to Hector for catching this typo!

Also changed "crush" to "stomp on" because that suits both the English and the Japanese better.